Note added on Timestamp column being null during backfill (#7652)
segmentio/segment-docs: src/unify/profiles-sync/tables.md

With raw tables, you have full control over the materialization of Profiles in your warehouse, as well as increased observibility.

Raw tables contain complete historical data when using historical backfill. 

> Note: `Timestamp` column will be empty for backfilled data. This is because we reverse engineer historical profile changes from the current state of the profile, and do not refer to the actual change history.

Author: ```timestamp``` refers to the instant at which the event wad ingested into Segment. For backfill, we don't refer to individual events, but to the final profile state. So we don't have access to the original event timestamps.

> Note: `Timestamp` column will be empty for backfilled data. This is because during backfill, we infer historical profile changes from the current state of the profile, and do not refer to the actual change history.

Reviewer: ```suggestion
The `Timestamp` column will be empty for backfilled data because, during backfill, historical profile changes are inferred from the current state of the profile and do not reflect the actual change history.
```

Slightly tweaked wording.
